src/main/java/mate/academy/util/HashUtil.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.util.Base64;
+
+public class HashUtil {
+    private static final String HASH_ALGORITHM = "SHA-256";
+    private static final SecureRandom RANDOM = new SecureRandom();
+
+    public static byte[] generateSalt() {
+        byte[] salt = new byte[16];
+        RANDOM.nextBytes(salt);
+        return salt;
+    }
+
+    public static String hashPassword(String password, byte[] salt) {
+        try {
+            MessageDigest md = MessageDigest.getInstance(HASH_ALGORITHM);
+            byte[] hash = md.digest(password.getBytes());
```

Comment:
The salt is not being used in the hashing process. You should combine the password and salt before hashing to ensure security. For example, use `md.update(salt);` before `md.digest(password.getBytes());`.